### PR TITLE
date-time visual tests and keros fixes

### LINKED ISSUE #6045: Calendar is missing visual tests
We have some tests for datepicker, but no dedicated, comprehensive coverage for datepicker.

@lorejoh12 is this something you're interested in adding? I'd be happy to give you a hand if you'd like. This will help to ensure that no visual changes occur in the datepicker due to child elements being tweaked.

### PR DESCRIPTION
**Pull request by lorejoh12**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #6045
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Adding visual tests for the date-time calendar. Also fixing some accessibility issues that were flagged by Keros/AccessibilityInsights.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8475)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
package.json was updated but the shrinkwrap wasn't.

File: apps/vr-tests/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "@types/webpack-env": "1.13.0",
     "@uifabric/prettier-rules": ">=1.0.1 <2.0.0",
     "@uifabric/tslint-rules": ">=1.0.1 <2.0.0",
+    "@uifabric/date-time": "0.3.3",
```
Comment (ecraig12345) [inline apps/vr-tests/package.json]: You should run `npm run rush-update` (and commit the result) to get this added to the shrinkwrap.

Comment (lorejoh12) [reply apps/vr-tests/package.json]: I knew I was missing something, thanks!

**Comment by khmakoto** [comment]
@OfficeDev/uifabric-calendar it would be great if one of you could take a look at this!

**Pull request merged**